src/helm/proxy/services/remote_service.py

```diff
@@ -27,6 +31,8 @@ class RemoteServiceError(Exception):
 
 
 class RemoteService(Service):
+    NOT_SUPPORTED_ERROR: str = "Not supported through the remote service."
```

Comment:
No action needed: We should eventually split all these miscellaneous things out of `Service`.